assets/test/SetExpiration.test.ts

```diff
@@ -23,10 +23,10 @@ test('Can set the expiration time', () => {
     }),
   );
 
-  wrapper.find('input[type="checkbox"]').simulate('click');
-  wrapper.find('.set_expiration--datetime').simulate('change');
-  wrapper.find('.react-datepicker-wrapper input').simulate('focus');
-  wrapper.find('.react-datepicker__day--015').simulate('click');
+  await user.click(screen.getByText('Schedule return to "Auto"'));
+  await user.click(screen.getByText('Date and time'));
+  await user.click(screen.getByRole('option', { name: /^Choose.*15th.*/ }));
```

Comment:
This is great! This is the kind of expressiveness that we should be aiming for in our UI test code.

Reply:
Yeah took a bit of fiddling to get it to work the way I wanted it to, but love how it turned out compared to how it was before!